timeboost/auctioneer.go

```diff
@@ -426,6 +475,33 @@ func (a *AuctioneerServer) resolveAuction(ctx context.Context) error {
 	return nil
 }
 
+func (a *AuctioneerServer) acknowledgeAllBids(ctx context.Context, round uint64) {
```

Comment:
This method could also be unit tested to verify that we delete from the unacked bids map for bids that are within the expected round

Reply:
This is now tested in TestConsumeNextBid_MultipleValidBids